third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -331,6 +331,28 @@ func resourceContainerCluster() *schema.Resource {
 				Default:  false,
 			},
 
+			"authenticator_groups_config": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+<% if version.nil? || version == 'ga' -%>
```

Comment:
Since the field was never part of the `google` provider, we can remove the `Removed` here and we can wrap this block entirely with `unless version == ga` (Checking for nullness isn't necessary either, we send a default in MM now)